types/setting.go

```diff
@@ -1455,7 +1455,7 @@ func ValidateAndUnmarshalToleration(s string) (*v1.Toleration, error) {
 
 	if strings.Contains(parts[0], "=") {
 		pair := strings.Split(parts[0], "=")
-		if len(parts) != 2 {
```

Comment:
Does this mean toleration always not working before?

Reply:
`parts` have already been checked above.
Ref: https://github.com/longhorn/longhorn-manager/blob/master/types/setting.go#L1446-L1448

Reply:
I see, so it could be just working coincidentally.